_data/rules.yml

```diff
@@ -223,6 +223,10 @@ groups:
                 description: '{{ $labels.instance }} interface {{ $labels.device }} has encountered {{ printf "%.0f" $value }} transmit errors in the last five minutes.'
                 query: 'increase(node_network_transmit_errs_total[5m]) > 0'
                 severity: warning
+              - name: Host filesystem readonly
+                description: '{{ $labels.instance }} partition {{ $labels.device }} has become readonly.'
+                query: 'node_filesystem_readonly{fstype!~"(tmpfs|proc|aufs|rpc_pipefs)"} == 1'
+                severity: error
```

Comment:
This should be either `warning` or `critical`.

Reply:
you are right,
`node_filesystem_readonly > 0 AND changes(node_filesystem_readonly[5m]) > 0
` 
the query  is ok, I've already tested.
I will close this PR, reopen a new one.